src/dvclive/live.py

```diff
@@ -191,13 +191,7 @@ def _init_dvc_file(self) -> str:
             if os.path.basename(self._dvcyaml) == "dvc.yaml":
                 return self._dvcyaml
             raise InvalidDvcyamlError
-        if self._dvc_repo is not None:
-            return os.path.join(self._dvc_repo.root_dir, "dvc.yaml")
-        logger.warning(
-            "Can't infer dvcyaml path without a DVC repo. "
-            "`dvc.yaml` file will not be written."
-        )
-        return ""
```

Comment:
>  dvcyaml: write to cwd instead of git root 

Looking at the diff, it seems it was dvc's root, not git root, right?

Reply:
Yes, you are right